### DISCUSSION TITLE
[6/7/2023] New Mojo Release!!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#341] [Announcements]
Hey everyone👋,

Playground has been updated with a new Mojo🔥 release ! Checkout the [change log](https://docs.modular.com/mojo/changelog.html) for what's changed since the previous release.
🙏 Kindly note, users with an active session in the Playground will need to restart their servers to get the updated notebook image.

Thanks!

**Comment by Jconsdad** [comment]
I have a small company and just ended my SAAS subscriptions and am in software limbo.  I am a boomer, who does not have a technical background, but did spend about 25,000 hours from 1985 to about 2012 on Foxpro for Dos 2.5 and 2.6.  Then I switched to Salesforce.com, Jitterbit Studio, Rackspace server and Apsona as a really useful overlay for SFDC.  

The other day I watched the Lex Fridman interview with Chris Lattner and then watched a video with Jeremy Howard where I think he suggested there would be a time where databases and programming could be in one program.

I am faced the prospect of creating and learning PostgreSql (DataGrip), Python (PyCharm (so far)) and GUI PyQt6.

I am just a small user, but I kind of got the feeling that if one could create tables and program the relationships and print to documents, pdfs, etc, going strait to Mojo, even if I have to wait, could save me a lot of time and be conceptually easier for a humble non-techie like me.  I have spent about 60,000 hours doing whatever it takes to learn to create and operate how my sees something, so I don't mind doing the work and taking time.

So the question is ... can you create relational databases in Mojo and just skip going through SQL ad Python?  I am assuming you can do the Gui and programing.

Much thanks in advance ... I am new to this.

**Reply by cjohnson318** [reply]
There's a couple of tools that allow you to access Postgres a lot easier than writing raw SQL. These are generally called ORMs, Object Relational Mappers(?). One option is to use SQLAlchemy, and another more opinionated/niche option might be to use the Django ORM.

I'm not sure how difficult it would be to do that via Mojo.

FWIW I typically use the following stack: Postgres for a database, Celery for scheduled tasks, Django and Django REST Framework for a web server and API layer, all wrapped in a Docker container, and hosted on DigitalOcean.

**Comment by Jconsdad** [comment]
Thanks so much!

I will look at your suggestions.

I saved all my tables in April.  Business is coming back but I don’t want to rush, I can manage the “old way” for a while.

Although I have been learning Data Grip and PyCharm, I haven’t started using them in production.

My needs aren’t that complex, but I have done this so long that I need to find to program to follow the way I think and not the other way around, if that makes any sense.

A friend of mine switched me to DigitalOcean when I left Rackspace.com.  

Again, thanks!



> On Jun 9, 2023, at 11:30 PM, Connor Johnson ***@***.***> wrote:
> 
> 
> There's a couple of tools that allow you to access Postgres a lot easier than writing raw SQL. These are generally called ORMs, Object Relational Mappers(?). One option is to use SQLAlchemy, and another more opinionated/niche option might be to use the Django ORM.
> 
> I'm not sure how difficult it would be to do that via Mojo.
> 
> FWIW I typically use the following stack: Postgres for a database, Celery for scheduled tasks, Django and Django REST Framework for a web server and API layer, all wrapped in a Docker container, and hosted on DigitalOcean.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/modularml/mojo/discussions/341#discussioncomment-6138170>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AJLN4TTIHJML6GO2RWWVJBDXKPS6FANCNFSM6AAAAAAY6YIOBQ>.
> You are receiving this because you commented.
> 

￼

**Reply by cjohnson318** [reply]
Best of luck! Feel free to reach out if you want another pair of eyes.
